package.json

```diff
@@ -218,6 +218,13 @@
           "command": "browser-preview.openSharedBrowser",
           "when": "false"
         }
+      ],
+      "explorer/context": [
+        {
+          "when": "explorerResourceIsFolder == false",
```

Comment:
You can add a `title` property to name the context menu item. See https://code.visualstudio.com/api/references/contribution-points#contributes.commands

Reply:
I tried and this does not work. Looks like vscode forces to be consistent in such action (context menu name to be the same as command name)
![image](https://user-images.githubusercontent.com/18731681/66887743-57d8f700-f01f-11e9-9545-a036cd26ecc8.png)

I possible workarounds is to create a menu entry from the code (not from package.json, not sure if it's possible) or create a hidden task with desired name (not sure if it's possible too)